# generated by patch-package 6.4.10
#
# command:
#   npx patch-package @lerna-lite/core
#
# declared package:
#   @lerna-lite/core: 1.5.1
#
diff --git a/node_modules/@lerna-lite/core/dist/utils/collect-updates/collect-updates.js b/node_modules/@lerna-lite/core/dist/utils/collect-updates/collect-updates.js
index c7cb766..28500a1 100644
--- a/node_modules/@lerna-lite/core/dist/utils/collect-updates/collect-updates.js
+++ b/node_modules/@lerna-lite/core/dist/utils/collect-updates/collect-updates.js
@@ -1,3 +1,4 @@
+
 "use strict";
 var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
@@ -28,7 +29,7 @@ function collectUpdates(filteredPackages, packageGraph, execOpts, commandOptions
     let committish = commandOptions.since;
     if ((0, has_tags_1.hasTags)(execOpts)) {
         // describe the last annotated tag in the current branch
-        const { sha, refCount, lastTagName } = (0, describe_ref_1.describeRefSync)(execOpts, commandOptions.includeMergedTags, gitDryRun);
+        const { sha, refCount, lastTagName } = (0, describe_ref_1.describeRefSync)({...execOpts, match: '*@*'}, commandOptions.includeMergedTags, gitDryRun);
         // TODO: warn about dirty tree?
         if (refCount === '0' && forced.size === 0 && !committish) {
             // no commits since previous release
diff --git a/node_modules/@lerna-lite/core/dist/utils/collect-updates/lib/has-tags.js b/node_modules/@lerna-lite/core/dist/utils/collect-updates/lib/has-tags.js
index f30b254..fa11bdf 100644
--- a/node_modules/@lerna-lite/core/dist/utils/collect-updates/lib/has-tags.js
+++ b/node_modules/@lerna-lite/core/dist/utils/collect-updates/lib/has-tags.js
@@ -1,3 +1,4 @@
+
 "use strict";
 var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
@@ -14,7 +15,7 @@ function hasTags(opts) {
     npmlog_1.default.silly('hasTags', '');
     let result = false;
     try {
-        result = !!(0, child_process_1.execSync)('git', ['tag'], opts);
+        result = !!(0, child_process_1.execSync)('git', ['tag', "--list", "*@*"], opts);
     }
     catch (err) {
         npmlog_1.default.warn('ENOTAGS', 'No git tags were reachable from this branch!');
